eras/babbage/impl/cardano-ledger-babbage.cabal

```
@@ -65,12 +65,12 @@ library
         bytestring,
         cardano-crypto-class,
         cardano-data >=1.0,
-        cardano-ledger-allegra >=1.1,
-        cardano-ledger-alonzo >=1.5.1,
+        cardano-ledger-allegra >=1.3,
+        cardano-ledger-alonzo >=1.6.0,
```

In bounds `0` is redundant
```suggestion
        cardano-ledger-allegra >=1.1,
        cardano-ledger-alonzo >=1.6,
```